cfgov/jinja2/v1/events/_macros.html

```diff
@@ -93,7 +93,7 @@
         {% endmacro %}
 
         {% macro _zip() %}
-            <span class="event-meta_zip p-postal-code">{{ zip }}</span>
+            <span class="event-meta_zip p-postal-code">{{ '%05d' | format(zip) }}</span>
```

Comment:
@kave what if the zip code doesn't contain a leading zero? Is this conditional somehow?